1.1/release_notes/sections/clause-1-intro.adoc

```diff
@@ -0,0 +1,98 @@
+== Introduction
+
+=== Scope
+
+The OGC GeoSPARQL standard supports representing and querying geospatial data on the Semantic Web. GeoSPARQL defines a vocabulary for representing geospatial data in RDF, and it defines extensions to the SPARQL query language for processing geospatial data. 
+
```

Comment:
I suggest changing "geospatial" to "spatial". There is not really an earth-based restriction in GeoSPARQL.

Reply:
I copied this from the [GeoSPARQL 1.1 spec introduction](https://opengeospatial.github.io/ogc-geosparql/geosparql11/spec.html#_introduction). Wording it differently in the release notes might be confusing!

Reply:
I'd agree. We either need to change it in both places, or leave as is.